src/fswAlgorithms/imageProcessing/opticalFlow/opticalFlow.cpp

```diff
@@ -170,33 +170,26 @@ void OpticalFlow::UpdateState(uint64_t CurrentSimNanos)
 }
 
 /*! This method reads an black and white image and makes a mask in order to remove the limb of the target.
- * First the contours are found and drawn on an image.
- * Second the main contour is filled with black
- * Third the image is blured and thresholded to remove roughness on the limb
- * Finally the edge is dilated to increase the margin off the limb
+ * First the distance transform is computed from the input Image to get the distance of each point from the
+ * dark (zero) pixel background.
+ * Second the image is thresholded
+ * Third the image is eroded by a masking value (eroding away N pixels from the edge of the limb)
+ * Finally the mask is converted to the correct type
  @return void
  @param inputBWImage cv::Mat of the input image
  @param mask cv::Mat of the output mask (binary black and white image)
  */
-void OpticalFlow::makeMask (cv::Mat const & inputBWImage, cv::Mat mask) const {
+void OpticalFlow::makeMask (cv::Mat const &inputBWImage, cv::Mat &mask){
```

Comment:
:large_blue_circle: Space between function name and opening brace.

Reply:
Done